x-pack/plugins/lists/kibana.json

```diff
@@ -1,10 +1,12 @@
 {
   "configPath": ["xpack", "lists"],
+  "extraPublicDirs": ["common"],
   "id": "lists",
   "kibanaVersion": "kibana",
   "requiredPlugins": [],
   "optionalPlugins": ["spaces", "security"],
+  "requiredBundles": ["securitySolution"],
   "server": true,
-  "ui": false,
+  "ui": true,
   "version": "8.0.0"
```

Comment:
As an aside, I think I saw somewhere we should be very shortly changing our version here to be kibanaVersion